src/eea/themecentre/browser/datareporting.py

```diff
@@ -62,12 +59,22 @@ def relatedReportingObligations(self):
             reso = res.getObject()
             for rodid in reso.reportingObligations:
                 if rodid not in rodsdone:
+
+                    try:
+                        rodinfo = rodsinfo[int(rodid)]
+                    except KeyError, err:
+                        # Obligation not found
+                        logger.exception(err)
+                        continue
+
                     rodsdone.append(rodid)
                     rodurl = rodbaseurl + rodid
+                    roddescription = rodinfo['DESCRIPTION']
```

Comment:
We should use `get` if possible in order to avoid `KeyError` exceptions

Reply:
fixed